themes/bootstrap3/templates/Recommend/ConsortialVuFind.phtml

```diff
@@ -0,0 +1,34 @@
+<div class="consortialVuFind">
+  <?php $results = $this->recommend->getResults() ?>
+  <?php $records = $results['records'] ?? [] ?>
+  <?php $resultCount = $results['resultCount'] ?? 0; ?>
+  <?php $moreResultsUrl = $this->recommend->getMoreResultsUrl(); ?>
+  <?php $intro = $this->translate('consortial_vufind_recommend_intro_html', [
+          '%%result_count%%' => $this->localizedNumber($resultCount),
+          '%%url%%' => $moreResultsUrl,
+  ]); ?>
+
+  <?php if (!empty($records)): ?>
+    <h2><?=$this->transEsc('consortial_vufind_recommend_heading')?></h2>
+    <?php /* \u200c is an empty translation value; see ExtendedIniReader.php. */ ?>
+    <?php if (!empty($intro) && ("\u{200c}" !== $intro)): ?>
```

Comment:
Turns out that @EreMaijala has already built a view helper to encapsulate this logic. See: https://github.com/NatLibFi/NDL-VuFind2/blob/dev/module/VuFind/src/VuFind/View/Helper/Root/TranslationEmpty.php

Reply:
Oh that's perfect, I swapped it in.  Since that view helper is now doing a second translation anyway, I also got rid of the $intro variable.

Reply:
Yeah, seems we can't get away from my original complaint about redundant lookups, but I guess I can't complain if we're doing things in a standard way. :-)